content/en/blog/2023/sig-instrumentation-spotlight.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: blog
+title: "Spotlight on SIG Instrumentation"
+slug: sig-instrumentation-spotlight
+date: 2023-01-31
+canonicalUrl: https://www.kubernetes.dev/blog/2023/01/31/sig-instrumentation-spotlight-2023/
+---
+
+**Author**: Imran Noor Mohamed(Delivery Hero)
+
+Observability requires the right data at the right time for the right consumer (human or piece of software) to make the right decision. In the context of Kubernetes, having best practices for cluster observability across all kubernetes components is crucial. 
+
+SIG Instrumentation helps to address this issue by providing best practices and tools that all other SIG uses to instrument kubernetes components like the *Kube API server*, *scheduler*, and *controller manager*.
+
+In this SIG Instrumentation spotlight, [Imran Noor Mohamed](https://www.linkedin.com/in/imrannoormohamed/), SIG Contribex-Comms tech lead talked with [Elana Hashman](https://twitter.com/ehashdn), and [Han Kang](https://www.linkedin.com/in/hankang), Chairs of SIG instrumentation, on how the SIG is organized, what are the current challenges and how anyone can get involved and contribute. 
+
+## About SIG Instrumentation
+
+**Imran (INM)**: Hello, thank you for the opportunity of learning more about SIG Instrumentation. Could you tell us a bit about yourself, your role, and how you got involved in SIG Instrumentation?
+
+**Han (HK)**: I started in SIG Instrumentation in 2018, and became a chair in 2020. I primarily got involved with SIG instrumentation due to a number of upstream issues with metrics which ended up affecting GKE in bad ways. As a result, we ended up launching an initiative to stabilize our metrics and make metrics a proper API.
+
+**Elana (EH)**: I also joined SIG Instrumentation in 2018 and became a chair at the same time as Han. I was working as a site reliability engineer (SRE) on bare metal Kubernetes clusters and was working to build out our observability stack. I encountered some issues with label joins where Kubernetes metrics didn’t match Kube-state-metrics (KSM) and started participating in SIG meetings to improve things. I helped test performance improvements to Kube-state-metrics and ultimately coauthored a KEP for overhauling metrics in the 1.14 release to improve usability.
+
+**Imran (INM)**: Interesting! Does that mean SIG-Instrumentation involves a lot of plumbing?
+
+**Han (HK)**: I wouldn’t say it involves a ton of plumbing, though it does touch basically every code base. We have our own dedicated directories for our metrics, logs, and tracing frameworks which we tend to work out of primarily. We do have to interact with other SIGs in order to propagate our changes which makes us more of a horizontal SIG.
+
+**Imran (INM)**: Speaking about interaction and coordination with other SIG could you describe how the SIG is organized?
```

Comment:
Was this actually:
```suggestion
**Imran (INM)**: Speaking about interaction and coordination with other SIGs could you describe how the SIG is organized?
```

Reply:
yes, that was the intention. I got confused between plural and singular forms.